Resources/Prototypes/_Floof/Research/civilianservices.yml

```diff
@@ -14,6 +14,20 @@
   technologyPrerequisites:
   - Hydroponics
 
+- type: technology
+  id: AdvancedTracking
+  name: research-technology-mini-hypo
+  icon:
+   sprite: Objects/Specific/Medical/implanter.rsi
+   state: implanter0
+  discipline: CivilianServices
+  tier: 2
+  cost: 15000
```

Comment:
I think 15k for a technology that hardly has any use for anyone is a bit overkill. Such technologies are typically made to cost 5k or 7.5k.